### PR TITLE
Test master

### PR DESCRIPTION
**Pull request by sheenaze**
This branch is deployed to the test server, I open this pull request so that you remember that they are pending changes  :)

### CODE REVIEW

**Comment by sheenaze** [comment]
And here are all the changes connected to finding an existing validation

**Comment by pstradio** [comment]
After testing the PR and talking to @sheenaze, we have established that the feature for re-running existing validations works as intended, in the sense that two validations with the exact same settings are matched and the message pops up allowing the user to decide whether to repeat the validation. The only remark (apparently, this was discussed before) is that the settings have to match to the exact decimals for spatial subsetting; this is very unlikely to happen e.g. for a manually defined bbox. 
A possible discussed solution to this issue is to look for approximately identical validations (to the first or second decimal) and to specify the settings of the previous validation in the prompted message, so that the user can decide whether to proceed. @wpreimes perhaps something to discuss on the next meeting?

File: validator/tests/test_validation.py [hunk 1]
```diff
@@ -518,49 +518,47 @@ def test_validation_ccia_ref(self):
...
-    #         new_run = ValidationRun.objects.get(pk=run_id)
-    #         print(new_run)
-    #         assert new_run
-
-    #         assert new_run.total_points == 15, "Number of gpis is off"
-    #         assert new_run.error_points == 0, "Too many error gpis"
-    #         assert new_run.ok_points == 15, "OK points are off"
-    #         self.check_results(new_run)
-    #         self.delete_run(new_run)
+    # @pytest.mark.long_running
```
Comment (wpreimes) [inline validator/tests/test_validation.py:521]: Forgot to uncomment this

Comment (sheenaze) [reply validator/tests/test_validation.py:521]: You mean the long running decorator?

Comment (wpreimes) [reply validator/tests/test_validation.py:521]: yes ;) but its fine, I will merge this PR last (after the cdf)

Comment (sheenaze) [reply validator/tests/test_validation.py]: done

**Comment by wpreimes** [comment]
also here make sure the latest pytesmo and qa4sm-reader version is used now

**Comment by sheenaze** [comment]
QA4SM reader is taken here directly from the repository, and if you want to merge cdf first, then set appropriate versions there and I'll just remove conflicts :)

**Comment by wpreimes** [comment]
looks like the old tests relied on the cdf matching not working...so maybe a small update to the tests is needed

**Comment by sheenaze** [comment]
But you want me to update those tests here?

**Comment by wpreimes** [comment]
Then again, it would have come up in the other branch already, so I think we can just use the new function...

**Comment by wpreimes** [comment]
> But you want me to update those tests here?

Lets see if it is necessary...

**Comment by wpreimes** [comment]
Looks good to me, @sheenaze you can merge this now if everything is ready

**Review by wpreimes — APPROVED** [review]

**Pull request merged**